src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -261,7 +255,8 @@ trait AttributeComponent {
               value_number double DEFAULT NULL,
               value_boolean bit(1) DEFAULT NULL,
               value_entity_ref bigint(20) unsigned DEFAULT NULL,
-              list_index int(11) DEFAULT NULL, """
+              list_index int(11) DEFAULT NULL,
+              list_length int(11) DEFAULT NULL, """
```

Comment:
I like nulls from a logical stand point when a field is optional or does not make sense is all instances. The list length only makes sense when it is a list.